cmdx.py

```diff
@@ -4096,9 +4192,38 @@ def createNode(self, type, name=None, parent=None):
         return DagNode(mobj, exists=False, modifier=self)
 
     @record_history
-    def parent(self, node, parent=None):
-        parent = parent._mobject if parent is not None else None
-        self._modifier.reparentNode(node._mobject, parent)
+    def parent(self, node, parent=None, preserve=True):
```

Comment:
Need to document what's going on here.

Reply:
oh right! i've seen you were planning to write the relative/absolute in another part of the code (def parent from root i remember) you really need that for rigging so i added that in the modifiers. i'm not sure if it's the best design and if we have to move that to the root command. you tell me

Reply:
Generally, I would leave convenience functionality to cmds (or PyMEL). There's no end to how much of cmds we could reimplement and it would not only bloat cmdx but also introduce unneccesary bugs for little to no gain.

I would move this into your own codebase, as a wrapper around this basic call.

Reply:
hmm right! i understand your point. feel free to drop it if you're not sure about it